geoips/commandline/geoips_config.py

```diff
@@ -0,0 +1,116 @@
+"""GeoIPS CLI "config" command.
+
+Various configuration-based commands for setting up your geoips environment.
+"""
+
+from os import listdir, environ
+from os.path import abspath, join
+import requests
+import tarfile
+
+from geoips.commandline.ancillary_info.test_data import test_dataset_dict
+from geoips.commandline.geoips_command import GeoipsCommand, GeoipsExecutableCommand
+
+
+class GeoipsConfigInstall(GeoipsExecutableCommand):
+    """Config Sub-Command Class for installing packages/data.
+
+    Supports installation of packages and test data needed for testing and/or running
+    your GeoIPS environment.
+    """
```

Comment:
Do we intend for this to be able to install more than just test_data in the future? If so, what else would it install?